opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -527,7 +534,7 @@ def _format_context(context):
         x_ctx["span_id"] = "0x{}".format(
             trace_api.format_span_id(context.span_id)
         )
-        x_ctx["trace_state"] = repr(context.trace_state)
+        x_ctx["trace_state"] = json.dumps(list(context.trace_state.items()))
```

Comment:
The output of `repr` is hard to deserialize. If it's just for debugging, I think JSON-serialized string is better to handle.